src/pages/VideoDetail.jsx

```diff
@@ -22,6 +27,14 @@ function VideoDetail() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [videoData]);
 
+  const checkInLiked = (id) => {
+    if (state.Liked.length > 0) {
+      console.log("once");
+      return state.Liked.find((item) => item._id === id);
+    }
+    return false;
+  };
+
   return (
     <div className="main-container">
       <SideNav />
```

Comment:
Suggestion - I see you have used SideNav across multiple components. If they are used in all the private routes, you could maybe add this as a commonly shared component.